content/articles/building-a-react-app-with-typescript/index.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /building-a-react-app-with-typescript/
+title: Building A React App With Typescript
+description: This article will give the reader a guide on how to build a React app with TypeScript.
+author: babatunde-koiki
+date: 2021-08-18T11:44-06
+topics: [React, Typescript]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/building-a-react-app-with-typescript/react-ts.png
+    alt: React App Using TypeScript
+---
+
+## Introduction
+
+![](react-ts.png)
+
```

Comment:
Check Your image paths

Reply:
Thanks for the feedback, I'd work on that now.